package.json

```diff
@@ -7,7 +7,9 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "preview": "vite preview",
-    "types": "tsc --noEmit"
+    "types": "tsc --noEmit",
+    "biome:ci": "biome ci",
```

Comment:
Do we need it?
I would keep `biome:check` and create `biome:write` which is the same with `--write` parameter.
The latter could be useful to format stuff before commit, if I understood how it works.
I'm currently using an IDE which does not support biome out of the box, differently from prettier. 

Also, is that notation with `:` a standard for npm script?

Reply:
Well, I guess we want to add some ci checks at a certain point no? In that case we should use biome ci.

Regarding the colon, it's not standard but it's common practice yeah, most projects do this.

Reply:
Also not a fan of precommit hooks hooks but yes, biome shines there because it's very fast

Reply:
-1 for commit hook.

We can still add the ci command later.
Would be useful for me the write command, since biome is not integraded in my IDE.

Not blocking though, we can edit those later.